rotkehlchen/api/rest.py

```diff
@@ -4200,3 +4191,39 @@ def get_all_evm_chains(self) -> Response:
                 for chain in ChainID
             ]),
         )
+
+    def get_ens_avatar(self, ens_name: str, match_header: Optional[str]) -> Response:
+        """
+        Searches for the ENS avatar of the given `ens_name`.
+        If found returns the avatar, otherwise returns 404 status code.
```

Comment:
you will need to update the docstring
```suggestion
        If found returns the a response with the avatar, otherwise a 404 empty response if the avatar is not found or a 409 empty response if there is any error
```